RHIDP-6667: Update TechDocs add-ons doc (#1026)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/assembly-techdocs-addons.adoc

//future release | Expand or collapse the subtitles in the TechDocs navigation menu and keep your preferred state between documentation sites.
|===

>  The preinstalled and external add-ons supported by {company} are exported to the TechDocs plugin by the `backstage-plugin-techdocs-module-addons-contrib` plugin package. This plugin package is preinstalled on {product} and enabled by default. The `disabled` status of the plugin package applies to all of the TechDocs add-ons that are exported by that package.

Reviewer: ```suggestion
The `backstage-plugin-techdocs-module-addons-contrib` plugin package exports both preinstalled and external add-ons supported by {company-name} to the TechDocs plugin. This plugin package is preinstalled on {product} and is enabled by default. If the plugin package is set to `disabled`, it applies to all TechDocs add-ons exported by the package.
```
